app/views/adventures/offline.html.erb

```diff
@@ -0,0 +1,42 @@
+<% content_for :theme, "#{@adventure.theme.capitalize}Theme" %>
+
+<div class="PlayerActions">
+
+<a href="#" id="saveBtn", class="SlantButton">Save</a>
```

Comment:
nitpick, but does this work as a `<button>`? typically preferred for any click action that's not a navigation to a page anchor or different url.

Reply:
I will try it out as a button now -- I used a link, because the other items in this div were `link_to` rails elements, so I was just keeping it consistent there. However, the other buttons on this page are hidden anyway, so I will change that